payments-core/src/test/java/com/stripe/android/view/CardInputWidgetTest.kt

```diff
@@ -104,6 +109,15 @@ internal class CardInputWidgetTest {
 
     @AfterTest
     fun cleanup() {
+        val application: Application = ApplicationProvider.getApplicationContext()
+        val activityInfo = ActivityInfo().apply {
+            name = CardInputWidgetTestActivity::class.java.name
+            packageName = application.packageName
+        }
+
+        val componentName = ComponentName(activityInfo.packageName, activityInfo.name)
+        shadowOf(application.packageManager).removeActivity(componentName)
```

Comment:
Is this still needed after adding the test rule?

Reply:
Oh right, removed it in the latest commit.